packages/aws-cdk-lib/aws-route53/README.md

```diff
@@ -247,6 +247,14 @@ route53.HostedZone.fromLookup(this, 'MyZone', {
 });
 ```
 
+The result of the `HostedZone.fromLookup()` operation will be written to a file
+called `cdk.context.json`. You must commit this file to source control so
+that the lookup values are available in non-privileged environments such
+as CI build steps, and to ensure your template builds are repeatable.
+To customize the cache key use the `additionalCacheKey` parameter.
+This can be useful if you want to scope the context variable to a construct 
+(eg, using `additionalCacheKey: this.node.path`).
+
```

Comment:
```suggestion

To customize the cache key use the `additionalCacheKey` parameter.
This can be useful if you want to scope the context variable to a construct 
(eg, using `additionalCacheKey: this.node.path`).

```ts
route53.HostedZone.fromLookup(this, 'MyZone', {
  domainName: 'example.com',
  additionalCacheKey: 'additionalKey',
});